docs/topics/land/land.md

```diff
@@ -0,0 +1,369 @@
+---
+description: Land
+---
+
+# Land
+
+## Introduction
+
+A LAND is a digital piece of real-estate in The Sandbox's metaverse. Game designers use LAND to build digital experiences, such as games or dioramas, and also populate it with ASSETs.
+
+<!-- add links to LAND, ASSET -->
+<!-- dioramas ? -->
+
+Each LAND is a unique, non-fungible ERC-721 token on the Ethereum blockchain. A total of 166,464 LANDs are available.
+
+Learn more about LAND in The Sandbox Metaverse in this [article](https://sandboxgame.gitbook.io/the-sandbox/land/what-is-land).
+
+## Model
+
+The Land token is an ERC721 token, in which the tokens represent pieces of the Sandbox world map. In total there are 166,464 LANDs, mapped to a 408x408 grid.
+
+### Features of the contract
+
+|              Feature | Link                                   |
+| -------------------: | :------------------------------------- |
+|            `ERC-165` | https://eips.ethereum.org/EIPS/eip-165 |
+|            `ERC-721` | https://eips.ethereum.org/EIPS/eip-721 |
+|   `ERC-721 metadata` |                                        |
+|            `ERC2771` |                                        |
+|        `Upgradeable` | No                                     |
+|             `Minter` | Yes                                    |
+| `WithSuperOperators` | Yes                                    |
+|                      |                                        |
+
+<!-- this wasn't finished. i couldn't find the IERC721Metadata in the code -->
+
+<!-- L: I couldn't find it either, this is the 0.5 version-->
```

Comment:
remove IERC721Metadata then
and update the table (i think it was based on the 0.8 one)